src/hardware-mapping.h

```diff
@@ -133,6 +133,11 @@ class HardwareMapping {
   // A value of 0 for 'motor' is accepted but does not connect it to anything.
   bool AddMotorMapping(LogicAxis axis, int motor, bool mirrored);
 
+  // Return the index of the first motor associated to an axis
+  // if the value is negative, the motor has not been mapped.
+  // i is the axis index
+  int GetMotorFromAxis(int i);
```

Comment:
Shouldn't the parameter be of LogicAxis type ? Also, I'd probably name it 'axis'.

Maybe 'from' should be 'for' ? After all, we are looking to get the motor that is for a particular axis.

Given that the function only returns a partial result, I'd call it GetFirstMotorForAxis().